client/state/analytics/actions.js

```diff
@@ -90,6 +91,18 @@ export const loadTrackingTool = trackingTool => ( {
 	},
 } );
 
+export const optOutTracks = isOptingOut => ( {
```

Comment:
Would we ever call this with `false`? If so `setOptOut` may be a more appropriate name.

If we never call this with `false`, I think we can drop the param, and use `optOutTracks`